tools/functional-tester/etcd-tester/checks.go

```diff
@@ -22,6 +22,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const retries = 3
```

Comment:
we need to check both rev and hash. we are able to do it anyway. then we can know that: OH... we thought rev is stablized but it is not... it caused hash to be unstable :(.